packages/devtools_app/lib/src/screens/performance/panes/rebuild_stats/rebuild_stats.dart

```diff
@@ -206,9 +210,16 @@ class _RebuildTableState extends State<RebuildTable> {
             ValueKey<String?>('${location.location.id}'),
         defaultSortColumn: _metricsColumns.first,
         defaultSortDirection: sortDirection,
-        onItemSelected: (item) {
-          // TODO(jacobr): navigate to selected widget in IDE or display the
-          // content side by side with the rebuild table.
+        onItemSelected: (item) async {
+          final location = item?.location;
+          if (location?.path != null) {
+            await _service?.postEvent('ToolEvent', 'navigate', <String, Object>{
+              'fileUri': location?.path ?? '',
```

Comment:
also nit: if the value of `location.path` is really a file URI, then the `path` field should be renamed to `fileUri`